cves/CVE-2013-0870.yml

```diff
@@ -361,11 +393,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
+CWE: 696
 CWE_note:
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Misplaced
```

Comment:
Nicknames are optional, but we prefer something a bit more catchy/creative.

Reply:
i wish i could be more creative😭